### PR TITLE
WIP: parsing alert transaction

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrumx/lib/atx.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    def _read_tx_and_hash_no_segwit(self):
+        return Deserializer.read_tx_and_hash(self)
+
+    def _check_if_alert_exist(self):
+        if self.binary_length > self.cursor:
+            return True
+        return False
+
+    def _check_if_alert_is_seg_wit(self):
```
Comment (mtwaro) [inline electrumx/lib/atx.py:16]: It's a convention to spell `segwit` together, not as `seg-wit`.

File: electrumx/lib/atx.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    def read_tx_block(self):
+        read = self._read_tx_and_hash_segwit
+        tx_no = self._read_varint()
+        tx = [read() for _ in range(tx_no)]
+
+        atx = []
+        if self._check_if_alert_exist():
+            atx_no = self._read_varint()
+            if not self._check_if_alert_is_seg_wit():
```
Comment (mtwaro) [inline electrumx/lib/atx.py:28]: Same as in line 16

File: electrumx/lib/coins.py [hunk 3]
```diff
@@ -3340,3 +3342,7 @@ class BitcoinVaultRegTest(Coin):
     ESTIMATE_FEE = 0.00001
     RELAY_FEE = 0.00001
     DAEMON = daemon.FakeEstimateFeeDaemon
+
+
+class BTCVAlert(BitcoinVaultRegTest):
```
Comment (mtwaro) [inline electrumx/lib/coins.py:3347]: Name of the coin should be BitcoinVault3Keys

File: electrumx/lib/atx.py (hunk 1 above)
Comment (kj-crypto) [reply electrumx/lib/atx.py]: done

File: electrumx/lib/atx.py (hunk 2 above)
Comment (kj-crypto) [reply electrumx/lib/atx.py]: done

File: electrumx/lib/coins.py (hunk 3 above)
Comment (kj-crypto) [reply electrumx/lib/coins.py]: done

**Pull request merged**